docs/editor/debugging.md

```diff
@@ -319,6 +319,27 @@ Source maps can be generated with two kinds of inlining:
 
 VS Code supports both the **inlined source maps** and the **inlined source**.
 
+#### JavaScript Source Map Tips
+

```

Comment:
Do you know what these sample projects are actually used for? I don't see where they are linked. And they might need a little work - https://github.com/Microsoft/vscode-samples/tree/master/node-express-typescript